### PR TITLE
fix re_building_group_disposal missing fk_columns

### PR DESCRIPTION
**Pull request by sjib**
Should fix the following issues:

- [x] https://github.com/teksi/wastewater/issues/47 - missing fk_columns
- [x] change obj_id to id as uuid - see https://github.com/teksi/wastewater/issues/47#issuecomment-1844883303 
- [x] also adapt obj_id to id as uuid for re_maintenance_event_wastewater_structure

### CODE REVIEW

**Comment by sjib** [comment]
@3nids Can you check on this
`Run docker compose exec db init_db.sh wait
time="2023-12-11T15:13:35Z" level=warning msg="The \"UID\" variable is not set. Defaulting to a blank string."
time="2023-12-11T15:13:35Z" level=warning msg="The \"GID\" variable is not set. Defaulting to a blank string."
waiting for postgres… 🐘
waiting for Postgres… 🐘
Error: Process completed with exit code 137.`

**Comment by 3nids** [comment]
Can we separate things?

This PR is to fix missing columns in the join tables but many views are added, could this be move to a separate PR if these changes are required (and if you create new join views, please create them with the python approach)

**Comment by sjib** [comment]
> many views are added
I integrated those view from master - I can try to kick them out again.

File: datamodel/changelogs/0001/09_tww_dictionaries.sql [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
 ------ with 3D coordinates
 
+DROP VIEW IF EXISTS tww_sys.dictionary_value_list;
+
+CREATE VIEW tww_sys.dictionary_value_list AS
+  SELECT p.relname AS vl_name, vl.*
+  FROM tww_vl.value_list_base vl, pg_class p
+  WHERE vl.tableoid = p.oid;
+
+
```
Comment (3nids) [inline datamodel/changelogs/0001/09_tww_dictionaries.sql:13]: this should not be here

**Comment by sjib** [comment]
Thanks @3nids

**Pull request merged**